doc/sslang-lrm.tex

```diff
@@ -525,6 +525,7 @@ \section{Expressions}
 \alt "if" <expr_blk> "{" <expr> "}" <expr_else>$^{?}$
 \alt "while" <expr_blk> "{" <expr> "}"
 \alt "fun" <pat>$^{+}$ "{" expr "}"
+\alt "match" <expr_blk> "{"< match_arms> "}"
```

Comment:
In this grammar section, you are missing the definition for `match_arms`. I think you missed one of my comments from last night. Notice what I did for `wait` and `par`. I folded the definition of `par_arms` directly into `par` and `wait` using regex notation.

Reply:
Done